reducers/views/rhs.js

```diff
@@ -113,6 +124,26 @@ function searchTerms(state = '', action) {
     }
 }
 
+function searchType(state = '', action) {
```

Comment:
Can you elaborate on the difference between the search type and the initial search type? Wondering if we can simplify this paradigm somehow.

Reply:
The initialSearchType was something that was never used. I removed it from the code. Thanks for catch that.